index.js

```diff
@@ -39,27 +39,22 @@ function getDirPath(filePath) {
 }
 
 function getDependencies(filePath, fileContents) {
-  const dependencies = [];
-
-  let imports;
   try {
-    imports = SolidityParser.parse(fileContents, "imports");
+    return SolidityParser.parse(fileContents, "imports")
+      .map(dependency => getNormalizedDependencyPath(dependency, filePath));
   } catch (error) {
     throw new Error(
       "Could not parse " + filePath + " for extracting its imports."
     );
   }
+}
 
-  for (let dependency of imports) {
-    if (dependency.startsWith("./") || dependency.startsWith("../")) {
-      dependency = getDirPath(filePath) + path.sep + dependency;
-      dependency = path.normalize(dependency);
-    }
-
-    dependencies.push(dependency);
+function getNormalizedDependencyPath(dependency, filePath) {
```

Comment:
This is really not necessary, sorry for the harder review

Reply:
I like the functional style beter :)